src/server/middlewares/helmet.ts

```diff
@@ -32,15 +27,12 @@ const DATA = ["data:"];
 
 export function configureHelmet(server: Express): void {
   server.use((_req, res, next) => {
-    const nonce = crypto.randomBytes(8).toString("hex");
+    const nonce = crypto.randomBytes(16).toString("base64");
```

Comment:
can you explain why this change was needed?

Reply:
Yep, this is just an optional change for a more secure nonce, but I can put it back to the way it was.